deepes/test/torch_agent/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required (VERSION 2.6)
```

Comment:
Why do we have another CmakeList.txt here? Can we use the prior CmakeLists.txt?

Reply:
done